doc/rados/operations/health-checks.rst

```diff
@@ -1694,38 +1694,39 @@ Stretch Mode
 INCORRECT_NUM_BUCKETS_STRETCH_MODE
 __________________________________
 
-Stretch mode currently only support 2 dividing buckets with OSDs, this warning
-suggests that the number of dividing buckets is not equal to 2 after stretch
-mode is enabled.  You can expect unpredictable failures and MON assertions
+Stretch mode currently only supports two dividing CRUSH buckets with OSDs. This warning
```

Comment:
supports only

mnemonic: ! "boldly go"

I wonder though if this is still true, as I know of someone trailing a 3-site stretch cluster. @gregsfortytwo @kamoltat ?